### DISCUSSION TITLE
YouTube Overheating in Librewolf 108.0.1

### DISCUSSION THREAD

**Discussion by stnert** [uBlockOrigin/uAssets#16363] [Q&A]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

I noticed that whenever I am on YouTube, uBlock, as I mentioned, on default settings, is giving overheating spikes, reaching about 95 degrees (Celsius).

My hardware settings are:

CPU: Intel i5-2520M (4) @ 3,200GHz
CPU: Intel 2nd Generation Core Processor Family
RAM: 4GB 
Disk: 240GB Kingston SSD 
OS: NixOS 22.11 Stable 

### URL(s) where the issue occurs.

`https://www.youtube.com/`

### Screenshot(s)

![Screenshot from 2023-01-16 17-18-01](https://user-images.githubusercontent.com/48295298/212759813-ea967bfe-584b-499b-becb-1014cf2f64d7.png)

### uBO version

1.46.0

### Browser name and version

LibreWolf 108.0.1

### Settings

[my-ublock-backup_2023-01-16_20.21.35.txt](https://github.com/uBlockOrigin/uAssets/files/10429257/my-ublock-backup_2023-01-16_20.21.35.txt)

### Notes

I tested it by reducing the quality to 720p and it still remained at the high temperature of 93 degrees (Celsius). 

Note: My hardware is in need of an internal cleaning (change of thermal paste and use of isopropyl alcohol), however, I came to the conclusion to be a bug in uBlock, because I also use in other browsers, the extension similar to uB0 and the same has not had this problem, so I ask you to analyze the case carefully. 

Thanks

**Comment by mapx-** [comment]
You should do all tests in the same browser, keeping only uBO (or the other extension .. which one ?) enabled and using only the default settings.

**Comment by mapx-** [comment]
@gorhill

**Comment by stephenhawk8054** [comment]
I'm also using Librewolf but on windows. However I don't see any heating issues with it. Without `Troubleshooting Information` (click on [Report button (chat icon)](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) to see it) and [profiling data](https://github.com/uBlockOrigin/uBlock-issues#performance-issues), it's difficult to investigate.

**Comment by Nojuuu** [comment]
Have you tried turning off ambient mode?

**Comment by garry-ut99** [comment]
Another similiar report https://github.com/uBlockOrigin/uBlock-issues/issues/2459
